include/bio/var_io/header.hpp

```diff
@@ -163,6 +166,7 @@ class header
      * \{
      */
     std::string              file_format = "VCFv4.3"; //!< The file format version.
+    std::string              file_date = transTime(); //!< The file date.
```

Comment:
Having file_date as an extra member seems like a good thing! However, I am not sure if I want it to be set automatically, because this results in output not being reproducible by default, which is really annoying when you are checking whether output has changed or not.

Related to this question is also whether BIO should add a line that says that the file was created with BIO. On the one hand, I would like this (advertising for the library); on the other hand, I am often annoyed by bcftools adding so many of those lines.

Reply:
So another solution would be to add a field which will be set by the user, e.g. the transTime() function would move to iGenVar. Would also be fine for me.

To your related question, I understand the desire to promote the library, but the important question should be whether it helps the user. At first I thought no, because why do I need to know who created it, since I am interested in the results from the tool itself. But then I thought if I don't understand a VCF line or there are errors in it (something like that) I might want to contact BIO instead of the tool creator that just uses the library. Hmm...